docs/style/introduction.md

```diff
@@ -1,2 +1,124 @@
+# Welcome to the BookBrainz Style Guidelines!
+=============================================
+## Introduction
+---------------
+How should I write edits in BookBrainz? Should I write edits according to the book cover?    
 
+Dear BookBrainz editors, worry less and just follow our (simple, yet concise) style guidelines!
+
+## When Should I Capitalize?
+----------------------------
+1. The **first** and **last words** 
+    + Always capitalized, regardless of the word length.
+
+2. [Adjectives](https://en.wikipedia.org/wiki/Adjective "Adjective Definition") 
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+
+3. [Adverbs](https://en.wikipedia.org/wiki/Adverb "Adverb Definition")
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+     
+4. [Articles](https://en.wikipedia.org/wiki/English_articles "Article Definition")
+    + Should **_only_** be capitalized when it is used as the first or last words.
+    + Regardless of the word length.
+    
+5. [Conjunctions](https://en.wikipedia.org/wiki/Conjunction_%28grammar%29 "Conjunction Definition")
+    + Always capitalized, no matter where it is used.
+    + **Except** [coordinate conjunctions](https://en.wikipedia.org/wiki/Conjunction_%28grammar%29#Coordinating_conjunctions) that should **_only_** be capitalized when it is used as the first or last words.
+    + Regardless of the word length.
+    
+6. [Demonstratives](https://en.wikipedia.org/wiki/Demonstrative "Demonstrative Definition")
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+
+7. [Infinitives](https://en.wikipedia.org/wiki/Infinitive "Infinitive Definition") 
+    + Should **_only_** be capitalized when it is used as the first or last words.
+    + Regardless of the word length.
+
+8. [Interjections](https://en.wikipedia.org/wiki/Interjection "Interjection Definition")
+    + Should **_only_** be capitalized when it is used as the first or last words.
+    + Regardless of the word length.
+
+9. [Interrogatives](https://en.wikipedia.org/wiki/Interrogative_word "Interrogative Definition")
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+    
+10. [Nouns](https://en.wikipedia.org/wiki/Noun "Noun Definition") 
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+    
+11. [Possessives](https://en.wikipedia.org/wiki/English_possessive "Possessive Definition")
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+
+12. [Prepositions](https://en.wikipedia.org/wiki/Preposition_and_postposition "Preposition Definition") that are more than five letters long
+    + Always capitalized, no matter where it is used.
+    + Regardless of the word length.
+
+13. [Pronouns](https://en.wikipedia.org/wiki/Pronoun "Pronoun Definition")
+    + **_Must_** be capitalized, no matter where it is used.
```

Comment:
This is inconsistent with other parts that are stated to always be capitalized (why is everything else just "always" but pronouns "must"?), should probably be made consistent.